File: LacesAPI/LacesViewModel/Request/AddImageRequest.cs
namespace LacesViewModel.Request
{
    public class AddImageRequest : LacesRequest
    {
        public int AssociatedEntityId { get; set; }
        public ImageInfo ImageInfo { get; set; }
    }

    public class ImageInfo
    {
        public byte[] ImageData { get; set; }
        public string FileFormat { get; set; }
    }
}

File: JavaHttpClasses/Request/AddImageRequest.java
public class AddImageRequest extends LacesRequest
{
	private int AssociatedEntityId;
	private ImageInfoRequest ImageInfo;
	
	public int getAssociatedEntityId()
	{
		return AssociatedEntityId;
	}
	
	public ImageInfoRequest getImageInfo()
	{
		return ImageInfo;
	}
	
	public void setAssociatedEntityId(int AssociatedEntityId)
	{
		this.AssociatedEntityId = AssociatedEntityId;
	}
	
	public void setImageInfo(ImageInfoRequest ImageInfo)
	{
		this.ImageInfo = ImageInfo;
	}
}